org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessParameterPanel.java

```diff
@@ -951,6 +952,8 @@ private void dynamicDisplay() {
 				}
 			} else if (editor.isVisible()) {
 				editor.setVisible(false);
+				rowList.get(i).setVisible(false);
+				
```

Comment:
you can't use the existing rowList variable for this - that's for field group and it will be made empty when field group change. you need to add a new instance variable to hold all rows.